_data/faq.yml

```diff
@@ -115,10 +115,126 @@ it:
+      acquired a right of use license. Only the software belonging to the first
+      category is subject to art. 69 of the CAD on reuse: for such solutions
+      administrations must obligatorily acquire full ownership of the source code
+      and must proceed with its release in open source. Instead the commercial
+      software acquired by means of a use right license can not be put to reuse.
+  item5:
+    question: Which are the indications provided for the purposes of compliance
+      of art. 69 regarding the choice of the license?
+    answer: |
+      The "Guidelines on the acquisition and reuse of software for the public
+      administrations", adopted by AgID in implementation of Articles 68 and 69
+      of the CAD, foresee that the choice of the license by a PA is
+      made by choosing a license among those certified by the Open Source
+      Initiatives. For more information about licensing see <a
+      href="https://docs.italia.it/italia/developers-italia/gl-acquisition-and-reuse-software-for-pa-docs/en/stabile/software-reuse/open-licences-and-selecting-a-licence.html#choosing-a-licence">this
+      resource</a>.
+  item6:
+    question: Is it possible in the process of Reuse of a solution
+      apply, by virtue of the principle of compatibility of licenses, to the work
+      derived a different license from the original work?
+    answer: |
+      The "Guidelines on the acquisition and reuse of software for the public
+      administrations", adopted by AgID in implementation of Articles 68 and 69
+      CAD, define, in paragraph 3.5.3, the decision tree for the
+      choice of an open license; in particular, if the release of the software
+      refers to a modification of existing open source software (so
+      taken for reuse by other administrations or owned by third parties)
+      the administration must use the same license with which the software has
+      been originally distributed, in order to facilitate its maximum
+      interoperability and reuse with other users of the same software.
+  item7:
+    Question: How can I report the non-application of articles 68 and 69 of the CAD?
+    answer: |
+      According to art. 17, paragraph 1-quater is established by the AgID the
+      Office of the Digital Ombudsman who has the task of collecting all reports
+      of supposed violations of the Code of the Digital Administration, or any
+      other rule on the subject of digitization and innovation, to guarantee
+      the digital rights of the citizens and businesses.
+      Anyone can report supposed violations of CAD provisions,
+      among which those ex art. 68 and 69, accessing the appropriate <a
+      href="https://difensorecivicodigitale.gov.it/index.php?a=add">area
+      dedicated</a> on AgID website.
+  item8:
+    question: What fines could be imposed at a PA that acquires solutions,
+      without making a technical and economic comparison?
+    answer: |
```

Comment:
I cannot understand the following lines, please review them if possible.

Reply:
My original translation was: 

**What sanctions could be imposed on a PA if it acquires solutions, software without carrying out the technical and economic comparative evaluation?**

I've tried to simplify it without losing the original meaning

Reply:
Sorry, I was referring to lines 215 and 216.